### ISSUE TITLE
How to pipe a subprocess' output?

### ISSUE THREAD

**Issue by cevr** [vadimdemedes/ink#342]
Hi! This project seems amazing.

I have a use case where I want to build a CLI on top of [lerna](https://github.com/lerna/lerna), so ideally I would like to be able to set a Box where the output is whatever lerna spits out.

Is there a recipe to do this with Ink?

Thank you!

**Comment by mAAdhaTTah** [comment]
I don't think you have to do anything special. Just save the stdout to an array or something & render it. The output from `stdout` I think can be treated as data like anything else.

**Comment by cevr** [comment]
@mAAdhaTTah Thanks, I'll try that!

**Comment by nitsanavni** [comment]
@cevr would be cool to see the result when you got it :)

**Comment by vadimdemedes** [comment]
@mAAdhaTTah is correct, as long as you get output text from the sub process, you can render it inside `<Text>`.

**Comment by cevr** [comment]
@vadimdemedes What about preserving the output formating (like color)?

**Comment by vadimdemedes** [comment]
I think it should work too.
